src/openrct2-ui/ride/Construction.cpp

```diff
@@ -24,6 +24,209 @@ using namespace OpenRCT2::TrackMetaData;
+        TrackElemType::LeftFlyerLargeHalfLoopInvertedUp,
+        TrackElemType::LeftFlyerLargeHalfLoopUninvertedDown,
+        TrackElemType::RightFlyerLargeHalfLoopUninvertedUp,
+        TrackElemType::RightFlyerLargeHalfLoopInvertedDown,
+        TrackElemType::RightFlyerLargeHalfLoopInvertedUp,
+        TrackElemType::RightFlyerLargeHalfLoopUninvertedDown,
+        TrackElemType::MultiDimInvertedFlatToDown90QuarterLoop,
+        TrackElemType::Up90ToInvertedFlatQuarterLoop,
+        TrackElemType::InvertedFlatToDown90QuarterLoop,
+        TrackElemType::MultiDimUp90ToInvertedFlatQuarterLoop,
+        TrackElemType::MultiDimFlatToDown90QuarterLoop,
+        TrackElemType::MultiDimInvertedUp90ToFlatQuarterLoop,
+        kSeparator,
+
+        // Twists
+        TrackElemType::LeftTwistDownToUp,
+        TrackElemType::LeftTwistUpToDown,
+        TrackElemType::RightTwistDownToUp,
+        TrackElemType::RightTwistUpToDown,
+        TrackElemType::LeftFlyerTwistUp,
+        TrackElemType::LeftFlyerTwistDown,
+        TrackElemType::RightFlyerTwistUp,
+        TrackElemType::RightFlyerTwistDown,
+        kSeparator,
+
+        // Barrel Rolls
+        TrackElemType::LeftBarrelRollUpToDown,
+        TrackElemType::LeftBarrelRollDownToUp,
+        TrackElemType::RightBarrelRollUpToDown,
+        TrackElemType::RightBarrelRollDownToUp,
+        kSeparator,
+
+        // Zero-G Rolls
+        TrackElemType::LeftZeroGRollUp,
+        TrackElemType::LeftZeroGRollDown,
+        TrackElemType::RightZeroGRollUp,
+        TrackElemType::RightZeroGRollDown,
+        TrackElemType::LeftLargeZeroGRollUp,
+        TrackElemType::LeftLargeZeroGRollDown,
+        TrackElemType::RightLargeZeroGRollUp,
+        TrackElemType::RightLargeZeroGRollDown,
+        kSeparator,
+
+        // Corkscrews
+        TrackElemType::LeftCorkscrewUp,
+        TrackElemType::LeftCorkscrewDown,
+        TrackElemType::RightCorkscrewUp,
+        TrackElemType::RightCorkscrewDown,
+        TrackElemType::LeftFlyerCorkscrewUp,
+        TrackElemType::LeftFlyerCorkscrewDown,
+        TrackElemType::RightFlyerCorkscrewUp,
+        TrackElemType::RightFlyerCorkscrewDown,
+        TrackElemType::LeftLargeCorkscrewUp,
+        TrackElemType::LeftLargeCorkscrewDown,
+        TrackElemType::RightLargeCorkscrewUp,
+        TrackElemType::RightLargeCorkscrewDown,
+        kSeparator,
+
+        // Dive loops
+        TrackElemType::LeftEighthDiveLoopUpToOrthogonal,
+        TrackElemType::LeftEighthDiveLoopDownToDiag,
+        TrackElemType::RightEighthDiveLoopUpToOrthogonal,
+        TrackElemType::RightEighthDiveLoopDownToDiag,
+        kSeparator,
+
+        // Tower
+        TrackElemType::TowerBase,
+        TrackElemType::TowerSection,
+        kSeparator,
+
+        // Mini Golf pieces
+        TrackElemType::MinigolfHoleA,
+        TrackElemType::MinigolfHoleB,
+        TrackElemType::MinigolfHoleC,
+        TrackElemType::MinigolfHoleD,
+        TrackElemType::MinigolfHoleE,
```

Comment:
After thinking about it, these pieces are a bit similar to the special behavior pieces. Tower and Minigolf maybe could go after BrakeForDrop.